symbol.md

```diff
@@ -0,0 +1,181 @@
+# SYMBOLS#PROCS
+
+Thank you for accepting this mission, and the mission is nothing else but to add Symbols#procs to our Ruby toolchain . However, to achieve our mission, we need to have expertise of  **symbols** & **Procs** and then with their combined power we will take on the mission.
+
+If you have some difficulties with any of these concepts, do not worry, I made arrangements for you. [The Ruby_Newbie Guide to Symbols](http://www.troubleshooters.com/codecorn/ruby/symbols.htm), has exactly what you will need for symbols and [Ruby Explained: Blocks, Procs, and Lambdas, aka "Closures"](http://www.eriktrautman.com/posts/ruby-explained-blocks-procs-and-lambdas-aka-closures). Once you are well equipped, onward to the mission.
+
+Let's construct a class for our soldier object
+```ruby
+  class Soldier
+    def initialize(name, weapon)
+      @name = name
+      @weapon = weapon
+    end
+
+    def name
+      @name
+    end
+
+    def first_name
+      @name.split.first
+    end
+
+    def last_name
+      @name.split.last
+    end
+
+    def weapon
+      @weapon
+    end
+
+    def salary
+      500 * name.length
+    end
+
+    def grade_a_weapon?
+      weapon.length > 5
+    end
+  end
+
+  soldier = Soldier.new("John Doe", "AK-47")
+
+  # From our symbol armory
+  soldier.send(:name) #=> John Doe
+  soldier.send(:weapon) #=> AK-47
+  soldier.send(:salary) #=> 4000
+```
+_note that using the **(.)** notation also works but for the sake of our mission we use send_
+Next up, we'll create a collection of all soldiers that signed up for the mission
+
+```ruby
+  soldiers = []
+  soldiers << Soldier.new('Big Mike', "Bazooka")
+  soldiers << Soldier.new('Small John', "Pistol")
+  soldiers << Soldier.new('Lightening Larry', "Uzi")
+
+  # From our proc armory
+  proc = Proc.new { |soldier| puts "Hail #{soldier.name}"}
+  soldiers.each(&proc)
+  #=> Hail Big Mike
+  #=> Hail Small John
+  #=> Hail Lightening Larry
+
+```
+
+What if we need a collection of just the names of the soldiers, we could using the map function do
+
+```ruby
+  soldiers.map{ |soldier|  soldier.send(:name) } #=> ["Big Mike", "Small John", "Lightening Larry"]
+
+  # Using our proc armory
+  proc = Proc.new { |soldier| soldier.send(:name) }
+  soldiers.map(&proc) #=> ["Big Mike", "Small John", "Lightening Larry"]
+```
+
+With symbol#proc, we can simplify this further
+
+```ruby
+  soldiers.map(&:name) #=> ["Big Mike", "Small John", "Lightening Larry"]
+```
+
+Now you know why we needed to make those stops we made at symbols and procs as such the question now won't be what each components of the code mean but rather, **How they work together?**
```

Comment:
You missed `and` after `procs`. Maybe you can just end the sentence there before continuing with `As such`.